### ISSUE TITLE
Automate creating teams from CATME output

### ISSUE THREAD

**Issue by pconrad** [project-anacapa/anacapa-github-linker#174]
As an instructor, I can import team assignments from CATME (see issue for format) and then the tool will automatically assign create "teams" in the GitHub organization and assign users to those teams.

**Comment by ProbablyFaiz** [comment]
Revised: Implement a generic CSV team import that doesn't have to deal with messy CATME output.

**Comment by chundhau** [comment]
I don't think that this needs to be tied to CATME. CATME Is one tool that can be used to generate teams. We just need to decide on a simple CSV fomat, e.g., 

studentemail, teamname 

We could include an optional third item that grants the student permission as a "team maintainer," e.g., the student should be able to add members to the team. However, with this new system perhaps we no longer need to grant students this permission?

@pconrad @ProbablyFaiz, any comments?

**Comment by ProbablyFaiz** [comment]
@chundhau Agreed. I think iterating upon the feature we demoed during the meeting so it takes email instead of GitHub ID is the best approach.

**Comment by ProbablyFaiz** [comment]
Also, to address the second part of that: right now, we add all users to teams as maintainers of those teams. We can make this configurable if it's helpful.
